companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "disguise",
+	"name": "disguise",
+	"shortname": "d3-osc",
+	"description": "OSC plugin for Companion",
+	"version": "3.0.0",
+	"license": "MIT",
+	"repository": "git+https://github.com/bitfocus/companion-module-disguise.git",
+	"bugs": "https://github.com/bitfocus/companion-module-disguise/issues",
+	"maintainers": [
+		{
+			"name": "Dan Hall",
+			"email": "dan@qed-productions.com"
+		}
+	],
```

Comment:
If you're removing the original authors of the module and replacing it with your own, please make sure you're reaching out to the previous author to get their OK for you to take over their module and do a complete rewrite/removing their author credit.

Reply:
I agree that it is not ok to do this without reaching out to the original author for consent - I will gladly do so. Some of these files have had sections copied and pasted from my other efforts and this was inadvertent